packages/components/src/interfaces/native/Table.ts

```diff
@@ -50,7 +50,6 @@ const TableCellSchema = Type.Unsafe<TableCellProps>(
     attrs: Type.Optional(TableBaseCellSchema),
     content: Type.Array(
       Type.Union([
-        Type.Ref(DividerSchema),
```

Comment:
remove unused import `DividerSchema`

Reply:
Updated in ff787dfb48ef98b6ed626620874ccab632937dd8.